spec/features/user_creates_a_review_spec.rb

```diff
@@ -10,13 +10,13 @@
     click_on "Add a Review"
 
     fill_in "Title", with: "Pepperoni"
-    fill_in "Body", with: "text text"
+    fill_in "Body", with: "#{'x' * 50}"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.